### ISSUE TITLE
FactionsPlus errors

### ISSUE THREAD

**Issue by tehrenderr** [MarkehMe/FactionsPlus#209]
http://pastebin.com/FurmWxiL
http://pastebin.com/7LT1Z1hc
http://pastebin.com/eDwAH282

As you can see I tried different version of Factions and FactionsPlus. Nothing seems to work.

**Comment by MarkehMe** [comment]
2.8.8 and above is not yet supported

2.8.7beta should be fine to use, but it's all kinda broken a bit. Please be patient, i'm creating the project into seperate modules at the moment!

**Comment by tehrenderr** [comment]
I am using a custom version of factions 2.7.5, I'm afraid I can't use 2.8.7 beta. Here is the error I get for 2.7.5
http://pastebin.com/hRRG2TVP
When do you think you can have it fixed by? Sorry for the rush, I just planned on having my server reset soon.

**Comment by DraksterAU** [comment]
Guessing your using cauldron?

**Comment by DraksterAU** [comment]
If you are, you can use 2.8.10 with Markehmes Massivecoreforgefixes plugin :)

**Comment by tehrenderr** [comment]
No, as stated in the pastebin error log I am using paperspigot 1.8.8

**Comment by DraksterAU** [comment]
Righto. Why can't you go higher then? factions 2.8.x works fine with paper

**Comment by tehrenderr** [comment]
They have a lot of bugs which needs to be fixed and my current factions is custom so I can't just upgrade. Besides if you look, I posted multiple pastebins in which I tried highger faction versions and the plugin still didn't work.

**Comment by DraksterAU** [comment]
your version of massive core in the past bins is wrong. You need to use 2.8.8 with 2.8.8 factions.

**Comment by DraksterAU** [comment]
The one which uses 2.8.6 factions in the pastebins uses beta 3 not 4. The other two suffer from massive core version issues.

**Comment by tehrenderr** [comment]
Those pastebins were old. Iv'e literally tried every version, here I have the same factions and massivecore and I am using beta 4. 
http://pastebin.com/VbJeZwr8

**Comment by tehrenderr** [comment]
@MarkehMe @DraksterAU Do you guys know if you can get this fixed really soon? I really want to be able to use this plugin for my server reset.

**Comment by DraksterAU** [comment]
@tehrenderr I am not the dev here, and Mark has already told several people he is rewriting the plugin. So probably not.
You can install this after your reset though. so not a worry there.

@MarkehMe You planning on supporting >2.0 <2.8?

**Comment by tehrenderr** [comment]
I am planning on using Factions 2.7.5 I'm probably just going to have to get a dev to fix it then, I need the warps feature to be fully functional for my faction server.

**Comment by DraksterAU** [comment]
How about upgrading your 'custom' features in 2.7.5 to 2.8.x ....

Good luck finding another dev to code FactionsPlus for you lol.

**Comment by tehrenderr** [comment]
If you actually read this post you can see that I've tried using factions 2.8 just to test it and it still never worked. I don't need a dev to code a whole new factionsplus lol, I can just get them to fix the small issue themselves and make it compatible.

**Comment by DraksterAU** [comment]
@MarkehMe Is already doing that though, so whats the point?

Do what you like. (usubscribing from post)

**Comment by MarkehMe** [comment]
FactionsPlus is really a pain to compile (trust me I know), but my new plugins will replace these issues + be better 😄  I suggest just staying on the older version of Factions until these are available.

**Comment by MarkehMe** [comment]
As an update.. I've released Factions Framework, and I'm creating everything into modules soon. 😄 

Bukkit: http://dev.bukkit.org/bukkit-plugins/factionsframework/
GitHub: https://github.com/MarkehMe/FactionsFramework/wiki
SpigotMC: https://www.spigotmc.org/resources/factions-framework.22278/

**Comment by tehrenderr** [comment]
I'm confused on what the framework actually does?

**Comment by MarkehMe** [comment]
It lets anyone easily create a plugin for any version of factions

**Comment by MarkehMe** [comment]
See #210
